eng/common/scripts/Verify-Links.ps1

```diff
@@ -257,13 +280,30 @@ function CheckLink ([System.Uri]$linkUri, $allowRetry=$true)
         $linkValid = $false
       }
       else {
+
         if ($null -ne $statusCode) {
+
           # For 429 rate-limiting try to pause if possible
-          if ($allowRetry -and $_.Exception.Response -and $statusCode -eq 429) {
-            $retryAfter = $_.Exception.Response.Headers.RetryAfter.Delta.TotalSeconds
+          if ($allowRetry -and $responsePresent -and $statusCode -eq 429) {
+
+            $headersPresent = $_.Exception.psobject.Properties.name -contains "Headers"
```

Comment:
While this works it is a little long handed. If we want to require PS 7.1 we can use the `.?` operator https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_operators?view=powershell-7.3#null-conditional-operators--and-

Reply:
I'm fine assuming >7.1 all the agents are already on 7.3 at this point.

Reply:
@weshaggard @benbp I don't think this works with strict mode v 3.0:

> Set-StrictMode -Version 1.0
> ${blah}?.omg
> Set-StrictMode -Version 3.0
> ${blah}?.omg
ParentContainsErrorRecordException: The property 'omg' cannot be found on this object. Verify that the property exists.